tests/ui-toml/max_suggested_slice_pattern_length/avoidable_slice_indexing_limit.rs

```diff
@@ -0,0 +1,23 @@
+#![deny(clippy::index_refutable_slice)]
```

Comment:
This file has the old lint name.

Reply:
This is the new lint name, otherwise it would also trigger the "unknown lint" lint :thinking:

Reply:
I mean the file name.